src/cluster_legacy.c

```diff
@@ -5704,7 +5704,7 @@ int verifyClusterConfigWithData(void) {
 
     /* Make sure we only have keys in DB0. */
     for (j = 1; j < server.dbnum; j++) {
-        if (kvstoreSize(server.db[j].keys)) return C_ERR;
+        if (server.db[j] && kvstoreSize(server.db[j]->keys)) return C_ERR;
```

Comment:
this condition will probably repeat many times. suggest we encapsulate in a function?

Reply:
introduced new function, `databaseEmpty`